docs/content/doc/advanced/external-renderers.en-us.md

```diff
@@ -0,0 +1,69 @@
+---
+date: "2018-11-23:00:00+02:00"
+title: "External renderers"
+slug: "external-renderers"
+weight: 40
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "advanced"
+    name: "External renderers"
+    weight: 40
+    identifier: "external-renderers"
+---
+
+# Custom files rendering configuration
+
+Gitea supports custom file renderings (i.e., Jupyter notebooks, asciidoc, etc.) through external binaries, 
+it is just matter of:
+* installing external binaries
+* add configuration to your `app.ini` file
+* restart your gitea instance
+
+## Installing external binaries
+In order to get file rendering through external binaries, their associated packages must be installed. 
```

Comment:
Tiny nit, there is inconsistencies between spacing in heading, some have a new line after and some don't.